server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/DBObjectNameAccessor.java

```diff
@@ -161,9 +161,9 @@ public Set<String> getPackageBodyNames() {
         if (session.getDialectType().isMysql()) {
             return Collections.emptySet();
         }
-        return accessor.listPackages(schema).stream()
```

Comment:
why change this?

Reply:
Because accessor#listPackages can only obtain packages and cannot obtain package bodiesBecause accessor#listPackages can only obtain packages and cannot obtain package bodies